libs/assertion/tests/unit/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ foreach(test ${tests})
   add_hpx_executable(${test}_test
                      SOURCES ${sources}
                      NOHPX_INIT NOLIBS
-                     DEPENDENCIES hpx_assertion
+                     DEPENDENCIES hpx_assertion hpx_datastructures
```

Comment:
Why is `hpx_datastructures` needed here?

Reply:
no need